core/types/crosslink.go

```diff
@@ -130,17 +130,25 @@ type CrossLinks []CrossLink
 // Sort crosslinks by shardID and then tie break by blockNum then by viewID
 func (cls CrossLinks) Sort() {
 	sort.Slice(cls, func(i, j int) bool {
-		return cls[i].ShardID() < cls[j].ShardID() ||
```

Comment:
Why change this part of the code?

Reply:
No need to change it in current pr, but i found bug and decided to fix it. Actually, it affects only CLs with equal BlockNum and different view id, i believe it's very rare case.